File: beecrowd/2764/2764 - Date Input and Output .py
while True:
    try:
        dd, mm, yy = map(int, input().split('/'))
        
        print(f"{mm:02d}/{dd:02d}/{yy:02d}")
        print(f"{yy:02d}/{mm:02d}/{dd:02d}")
        print(f"{dd:02d}-{mm:02d}-{yy:02d}")
        
    except EOFError:
        break

File: beecrowd/2764/2764 - Date Input and Output .cs
using System;


class Program
{
  static void Main(string[] args)
  {
    string input;
    string[] date;
    int dd, mm, yy;

    while ((input = Console.ReadLine()) != null)
    {
      date = input.Split('/');
      dd = int.Parse(date[0]);
      mm = int.Parse(date[1]);
      yy = int.Parse(date[2]);

      Console.WriteLine($"{mm:D2}/{dd:D2}/{yy:D2}");
      Console.WriteLine($"{yy:D2}/{mm:D2}/{dd:D2}");
      Console.WriteLine($"{dd:D2}-{mm:D2}-{yy:D2}");
    }
  }
}
